### PR TITLE
Additonal revert of #2877

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Additonal revert of #2877 

 

### Details

- Relate to #2885

### CODE REVIEW

**Review by akaszynski — APPROVED** [review]

**Review by banesullivan — APPROVED** [review]

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: "Pull Request Labeler"
 on:
   pull_request:
-    types: [opened, synchronize, reopened, labeled, unlabeled]
+    types: [opened, reopened]
   workflow_dispatch:
```
Comment (tkoyama010) [inline .github/workflows/label.yml:5]:
```suggestion
```

**Review by akaszynski — COMMENTED** [review]
Still LGTM.

**Comment by akaszynski** [comment]
@tkoyama010, since I've check and we indeed have the right permissions in [Workflow permissions
](https://github.com/pyvista/pyvista/pull/2885#issuecomment-1168022743), should we make sure this doesn't run on forks?

**Comment by tkoyama010** [comment]
@akaszynski No. Thanks and sorry for this trouble. I will try a dummy account to see if it works.

**Comment by adeak** [comment]
> @akaszynski No. Thanks and sorry for this trouble. I will try a dummy account to see if it works.

If my PRs are any indication, it's enough to open a PR from a branch on tkoyama/pyvista instead of pyvista/pyvista.

**Comment by tkoyama010** [comment]
@adeak Thanks. I forked it now https://github.com/tkoyama010/pyvista . Could you please open test Pull Request with "document" label?

**Comment by adeak** [comment]
Sure, I can do that, just ping me once this is merged (I'll probably forget).

But we should make sure we all mean the same thing. I took @akaszynski's comment about "running on forks" to refer to cases like https://github.com/pyvista/pyvista/pull/2885: someone opening a PR to `pyvista/pyvista` from a branch at `user/pyvista`. So what I meant was for you to do that: opening a proper pyvista PR, but instead of using a (test) feature branch on the `pyvista/pyvista` repo, use a branch on your fork (like I did with my fork at `adeak/pyvista`).

From what you said it seems you suggested that I open a PR to _your_ fork from _my_ fork. And now I'm not sure what @akaszynski meant exactly :smile: I can help with any configuration, assuming we three agree on a test setup.

**Review by akaszynski — APPROVED** [review]

**Comment by tkoyama010** [comment]
@adeak  Now I made a [forked repository](https://github.com/tkoyama010/pyvista).

**Comment by adeak** [comment]
@tkoyama010 I think @akaszynski tested this already in https://github.com/pyvista/pyvista/pull/2889.

**Pull request merged**